lib/File/Temp.pm

```diff
@@ -501,13 +502,16 @@ sub _gettemp {
       # Attempt to open the file
       my $open_success = undef;
       if ( $^O eq 'VMS' and $options{"unlink_on_close"} && !$KEEP_ALL) {
+        my $flags = $OPENFLAGS;
+        $flags = ($flags ^ O_RDWR) | O_WRONLY if $options{write_only};
```

Comment:
Why the xor?

Reply:
Wanted to turn off O_RDWR bit.  Looks like xor toggles/flips the bit, which does not necessarily clear it.  Will update to ($flags & ~O_RDWR) to clear it.